packages/@react-spectrum/icon/docs/Icon.mdx

```diff
@@ -25,12 +25,32 @@ category: Misc
 ---
 
 # Icon
-<p>Docs coming soon</p>
+
+<p>{docs.exports.Icon.description}</p>
 
 ## Available Icons
 
 ### Workflow
-<HeaderTable name={workflowIconPackageData.name} version={workflowIconPackageData.version} />
+
+<HeaderInfo
+  packageData={workflowIconPackageData}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/icons/'}
+  ]} />
+
+<br />
+
+Spectrum workflow icons are graphical metaphors or symbols that users interact with to navigate and manipulate objects.
+They all appear at the same size within each platform, 18 px for desktop and 22 px for mobile. See [ActionButton](/react-spectrum/ActionButton.html#content)
+and [Picker](/react-spectrum/Picker.html#complex-items) for some examples of icon usage within React Spectrum components.
+
+See the table below for a full list of available workflow icons. Note that you are free to rename the imports, meaning the following
+two are equivalent:
+
+```tsx
+import Add from '@spectrum-icons/workflow/Add;
+import IconAdd from '@spectrum-icons/workflow/Add';
```

Comment:
This is displaying wrong for some reason.
![image](https://user-images.githubusercontent.com/309540/85791049-10deed80-b6e6-11ea-89f1-39a5bf6a0cd0.png)

Reply:
Not sure it's totally necessary to show the imports here since we already show them all in the table...

Reply:
Its more for letting people know that they can rename the import if need be (mirroring the README for Icons that we have in repo)